.github/actions/release/run

```diff
@@ -166,5 +166,3 @@ cat "$section_wasm_verification" >> "$body" && echo >> "$body" && rm "$section_w
 cat "$body"
 
 echo "notes-file=$body" >> "$GITHUB_OUTPUT"
-
-[ -n "${GITHUB_ENV:-}" ] && echo "RELEASE_ACTION_BODY_FILE=$body" >>"$GITHUB_ENV"
```

Comment:
What was this for?

Reply:
This used to set the filename as an env variable in Github (see [here](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files)). But we don't actually use it, so I removed it.

Reply:
Are there no other occurrences of `RELEASE_ACTION_BODY_FILE`?